themes/classic/templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -23,7 +23,9 @@
  * International Registered Trademark & Property of PrestaShop SA
  *}
 {block name='product_miniature_item'}
-  <article class="product-miniature js-product-miniature" data-id-product="{$product.id_product}" data-id-product-attribute="{$product.id_product_attribute}" itemscope itemtype="http://schema.org/Product">
+<div itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+  <meta itemprop="position" content="{$position}" />
+  <article class="product-miniature js-product-miniature" data-id-product="{$product.id_product}" data-id-product-attribute="{$product.id_product_attribute}" itemprop="item" itemscope itemtype="http://schema.org/Product">
```

Comment:
@Progi1984 This is the reason why travis is failed, you add `<div itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">` before each `<article> `and thats why our selector is not valid anymore. 
We can change the selector in test  to `[itemprop="itemListElement"]:nth-of-type(2) .product-miniature a.product-thumbnail` to pass travis but I'm not sure that E2E test will pass (nightly tests)

Reply:
:ok: I understand. Do you want I modify it in my PR ? Thank you for your help

Reply:
It's better to not use xpath and only have `.product-miniature a.product-thumbnail`

Reply:
@PierreRambaud It's a CSS selector.
@Progi1984, are you able to tweak your markup to make the original selector work ? If not, we'll try to change the selector to be more agnostic of the markup.

Reply:
@SimonGrn No, the modification permits to be valid to the microdata Schema Product

Reply:
@SimonGrn My bad. I misread.
But anyway, we must no consider hierarchy level with css selector.

`.products :nth-of-type(2) a.product-thumbnail` => Should be better and independent

Reply:
> @boubkerbribri ok I understand. Do you want I modify it in my PR ? Thank you for your help

@Progi1984 , yes please, you can use simplified selector : `[itemprop="itemListElement"]:nth-of-type(2) a.product-thumbnail`

Reply:
@boubkerbribri Done